.coveragerc

```diff
@@ -1,15 +1,7 @@
 # .coveragerc to control coverage.py
 [run]
 omit =
- jmclient/jmclient/jsonrpc.py
- jmclient/jmclient/electruminterface.py
- jmclient/jmclient/btc.py
- jmclient/test/*
- jmclient/setup.py
- jmbitcoin/test/*
- jmbitcoin/setup.py
- jmbase/test/*
- jmbase/setup.py
- jmdaemon/test/*
- jmdaemon/setup.py
- jmdaemon/jmdaemon/socks.py
+ src/jmclient/jsonrpc.py
+ src/jmclient/electruminterface.py
+ src/jmclient/btc.py
+ src/jmdaemon/socks.py
```

Comment:
The last 3 no longer exist and won't in future, in the project, so I guess remove them.